bin/victory-cli.js

```diff
@@ -24,8 +23,6 @@ program.option(
   "'area', 'bar', 'line', 'scatter' or 'pie'",
   "line"
 );
-program.option("-f, --format [format]", "'png' or 'svg'", "png");
-program.option("-p, --print", "Prints chart to console (iTerm3 & .png format only!)");
```

Comment:
As this feature was dependent on a generated png, we no longer support printed imgs to iTerm either